test/transactions/0_send.js

```diff
@@ -199,64 +203,56 @@ describe('#send transaction', () => {
 			});
 		});
 
-		it('should create a send transaction', () => {
-			const sendTransactionWithoutSecondSecret = send({
-				recipientId,
-				amount,
-				secret,
-			});
-			sendTransaction.should.be.ok();
-			sendTransaction.should.not.be.equal(sendTransactionWithoutSecondSecret);
-		});
-
-		it('should create send transaction with second signature and data', () => {
+		it('should create a send transaction with data property', () => {
 			sendTransaction = send({
 				recipientId,
 				amount,
 				secret,
 				secondSecret,
 				data: testData,
 			});
-			sendTransaction.should.be.ok();
+
+			sendTransaction.asset.should.have.property('data');
 		});
 
-		describe('returned send transaction', () => {
-			it('should have second signature hex string', () => {
-				sendTransaction.should.have
-					.property('signSignature')
-					.and.be.hexString();
+		it('should have the second signature property as hex string', () => {
+			sendTransaction.should.have.property('signSignature').and.be.hexString();
+		});
+
+		describe('verification of the created transaction', () => {
```

Comment:
Do these tests belong here? Seems like they're testing `cryptoModule.verifyTransaction`. Maybe the more appropriate test would be to see if `signature` or `signSignature` is equal to the result of calling (a stubbed) `cryptoModule.signTransaction`.